api/api.go

```diff
@@ -5,16 +5,24 @@ import (
 	"github.com/trustwallet/blockatlas/platform"
 )
 
-func SetupPlatformAPI(root gin.IRouter) {
-	for _, api := range platform.Platforms {
-		RegisterCollectionsAPI(root, api)
-		RegisterTransactionsAPI(root, api)
-		RegisterCustomAPI(root, api)
-		RegisterTokensAPI(root, api)
-		RegisterStakeAPI(root, api)
+func SetupPlatformAPI(router gin.IRouter) {
```

Comment:
What is the purpose to split it?

Reply:
A small step in the direction when services are more separated, and handlers are not decided based on interface implementation or not, but explicit lists.  Staking and Naming (as there are only a few of them) have been completely separated.
I plan to change the routes one for each coin, then the for loops here will be gone.

Reply:
Cannot see any sense. You are just make more code by making 5 loops instead of 1

Reply:
It's about separation of concerns.  Think like this:

A: Get a bunch of handlers, DECIDE which ones are relevant and REGISTER an path for them
B: Get a list of relevant handlers, REGISTER a path for them.

Separation of concerns sometimes comes with a few extra lines, and I think it worth a few lines.
However, my plan was to change the routes generic w.r.t. coin, in which case the for loops would have disappear completely, but this turned to be not plausible (see prev. comment).  Therefor I may revert this change.